external/uapaotredist/project.json.template

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "frameworks": {
+    "uap10.1": {
```

Comment:
I can do that. The reason why I have it as uap10.1 is because the NugetTargetMoniker get's set automatically to uap10.1 when doint a uapaot vertical build, so that way I wouldn't have to manually set it in the .depproj, but I can change that.